test/Microsoft.AspNetCore.Mvc.FunctionalTests/HtmlGenerationTest.cs

```diff
@@ -530,6 +530,43 @@ public class HtmlGenerationTest :
             Assert.Equal(expected, response);
         }
 
+        [Fact]
+        public async Task ReregisteringAntiforgeryInBeginFormAndBeginRouteForm_DoesNotAddDuplicateAntiforgeryTokens()
```

Comment:
Please simplify this test name e.g. `MultipleFormElements_DoNotConfuseTheSystem` :trollface: or `MultipleFormsAndAntiForgeryTokens_AddOneTokenPerForm()`.